helpers/testing_helpers.go

```diff
@@ -0,0 +1,134 @@
+function automatically adds a connection to your local mondgb host.
+
+Parameters:
+- databaseName is the name of the database, which should be **unique**.
+- tester is a function that runs the test.
+
+Example:
+It("Test Description", helpers.TestWithMongo("controller-method", func() {
+	// test a controller that uses mongo/mgm
+	...
+}))
+*/
+func TestWithMongo(databaseName string, tester func()) func() {
+	return func() {
+		// First connect to the database.
+		database := ConnectTestDB(databaseName)
+
+		// Immediately reset the database. Even though we reset the database after
+		// the test runs, we still reset it before as well. This is because if a test
+		// fails halfway through, the reset afterwards is never hit.
+		ResetTestDB(database)
+
+		// Run the test and reset the database.
+		tester()
+		ResetTestDB(database)
+	}
+}
+
+/*
+Connects to a passed-in database on your LOCAL mongodb host, for testing purposes.
+
+To allow for concurrent testing specs, this function should be called
+**separately** on each test (that requires mongodb), with a **different** database name.
+For more information, see https://github.com/drift-org/backend/wiki/Testing
+
+Parameters:
+- databaseName is the name of the database.
+*/
+func ConnectTestDB(databaseName string) *mongo.Database {
+
+	// Ginkgo runs concurrent tests in different processes, so load the .env file each time.
+	err := godotenv.Load(os.ExpandEnv("../.env"))
+	Expect(err).NotTo(HaveOccurred())
+
+	// Parse the local database url.
+	databaseURL := os.Getenv("TESTING_MONGO_URL")
+	Expect(databaseURL).NotTo(BeEmpty(), "TESTING_MONGO_URL is missing from .env")
+
+	// Make sure that the word "localhost" is inside the databaseURL. We want to double check that this
+	// is a locally ran mongo client and make sure we aren't writing to our production database.
+	Expect(databaseURL).To(ContainSubstring("localhost"), "TESTING_MONGO_URL must contain 'localhost'")
+
+	// Create the client options. Since this test is in a different process, set the default config.
+	clientOptions := options.Client().ApplyURI(databaseURL)
+	err = mgm.SetDefaultConfig(nil, databaseName, clientOptions)
+	Expect(err).NotTo(HaveOccurred())
+
+	// Connect to the client.
+	client, err := mongo.Connect(context.TODO(), clientOptions)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(client).NotTo(BeNil())
+
+	return client.Database(databaseName)
+}
+
+/*
+Resets to a passed-in database on your LOCAL mongodb host, for testing purposes.
+
+This function should be called after each test that used mongodb has finished.
+We do this to make sure our tests are completely deterministic with no state
+prior to testing.
+*/
+func ResetTestDB(database *mongo.Database) {
+	database.Drop(mgm.Ctx())
+}
+
+/*
+Logs std output. The standard fmt.Println doesn't show up when testing with ginkgo.
+For debugging purposes ONLY.
+*/
+func GinkgoLog(message string) {
+	_, _ = fmt.Fprintf(GinkgoWriter, "[DEBUG]"+message)
```

Comment:
Super small (and optional) nit, but is _, _ really necessary? You can just run the fmt.Fprintf without that right